ts/features/lollipop/index.ts

```diff
@@ -90,11 +90,11 @@ export const handleRegenerateKey = (
         if (isMixpanelEnabled) {
           trackLollipopKeyGenerationFailure(error.message);
         }
-        dispatch(lollipopRemovePublicKey());
+        dispatch(lollipopRemoveEphermeralPublicKey());
```

Comment:
What about renaming `handleRegenerateKey` to, let's say, `handleRegenerateEphermeralKey` to suggest that we are dealing with the ephemeral key and not the login key? What do you think?